test/calibration/test_update_library.py

```diff
@@ -160,7 +160,7 @@ class TestDragUpdate(QiskitTestCase):
     def test_drag(self):
```

Comment:
Did you want to add a fine DRAG updater test?

Reply:
Yes, done here: ed62303